.changeset/many-crabs-cross.md

```diff
@@ -0,0 +1,11 @@
+---
+'@responsive-image/react': minor
+---
+
+Export the CSS as well as a JS only variant
+
+Most of the time the default export should "just work", but
+this change adds some flexibility for users of the package.
```

Comment:
Can you remove the explicit line breaks here, other than for actual paragraphs, please? This was causing some unexpected formatting in the created GitHub releases during publishing.